src/content/docs/features/email.md

```diff
@@ -2,11 +2,17 @@
 title: Email Handlers
 ---
 
-Membrane programs are capable of sending and receiving emails. Recieving an email works similar to how [HTTP endpoints](/features/endpoints/) are handled. All you need to do is export an `email` function from the root of your program and use the program's unique email address.
+Membrane programs are capable of sending and receiving emails. Recieving an email works similarly to how [HTTP endpoints](/features/endpoints/) are handled. All you need to do is export an `email` function from the root of your program and use the program's unique email address.
+
+<!-- TODO: create reference section for `sms` and link to that instead -->
```

Comment:
I have a ticket to add a reference section for all of the Membrane API: [T-685](https://membrane.height.app/T-685)